chapter_optimization/lr-scheduler.md

```diff
@@ -155,26 +246,11 @@ class SquareRootScheduler:
         return self.lr * pow(num_update + 1.0, -0.5)
 ```
 
-```{.python .input}
-#@tab tensorflow
-class SquareRootScheduler:
-    def __init__(self, lr=0.1):
-        self.lr = lr
-
-    def __call__(self, epoch):
-        return self.lr * pow(epoch + 1.0, -0.5)
-```
-
 Let us plot its behavior over a range of values.
 
 ```{.python .input}
-scheduler = SquareRootScheduler(lr=1.0)
-d2l.plot(np.arange(num_epochs), [scheduler(t) for t in range(num_epochs)])
-```
-
-```{.python .input}
-#@tab tensorflow
-scheduler = SquareRootScheduler(1.0)
+#@tab all
+scheduler = SquareRootScheduler(lr=0.3)
```

Comment:
Updated the LR value to 0.3. This is required for stable training which can converge for PyTorch implementation when calling the train function below.